plugins/org.jkiss.dbeaver.ext.databricks/src/org/jkiss/dbeaver/ext/databricks/DatabricksDataSource.java

```diff
@@ -51,4 +59,19 @@ private boolean isLegacyDriver() {
         return CommonUtils.equalObjects(DatabricksConstants.DRIVER_CLASS_LEGACY, getContainer().getDriver().getDriverClassName());
     }
 
+    @NotNull
+    @Override
+    protected Properties getAllConnectionProperties(
+        @NotNull DBRProgressMonitor monitor,
+        JDBCExecutionContext context,
+        String purpose,
+        DBPConnectionConfiguration connectionInfo
+    ) throws DBCException {
+        String userAgent = URLEncoder.encode(
+            GeneralUtils.getProductName() + "/" + GeneralUtils.getProductVersion(),
```

Comment:
It seems that here the company name is missing. There will be something like this: 
- DBeaver Ultimate/25.1.4
- CloudBeaver Enterprise/25.1.4

And according to Databricks docs, we need to pass something like this:
- DBeaver DBeaver Ultimate/25.1.4
- DBeaver CloudBeaver Enterprise/25.1.4